frontend/.vscode/settings.json

```diff
@@ -3,10 +3,10 @@
   "editor.formatOnSave": true,
   "editor.formatOnPaste": false,
   "editor.codeActionsOnSave": {
-    "source.organizeImports": false
+    "source.organizeImports": true
   },
   "[javascript]": {
-  "editor.formatOnSave": true
+    "editor.formatOnSave": true
```

Comment:
I think this line is'nt useful. (we already define a common rule on L3)
I see a diff here. You've try without this line but it doesn't work?

Reply:
I'm not changing this part, the diff is just because the previous version was not consistently formatted.
Actually, I'm not even sure it is taken into account now that `sirius-components` is in TypeScript instead of JS.
I'll have to check.